src/BloomExe/web/EnhancedImageServer.cs

```diff
@@ -272,6 +272,8 @@ protected override bool ProcessRequest(IRequestInfo info)
 			else if (localPath.StartsWith("node_modules/jquery/dist/jquery.js"))
 			{
 				localPath = BloomFileLocator.GetBrowserFile("jquery.min.js");
+				// Avoid getting "output/browser/" removed on Linux.
+				return ProcessContent(info, localPath);
```

Comment:
On a user-install, there will be no "output/browser", right? So it's not going to work if leave it there?